README.md

```diff
@@ -199,20 +211,74 @@ _Note: This will be different if you installed DVWA into a different directory._
 
 - - -
 
-## Docker Container
+## Troubleshooting
 
-_This section of the readme was added by @thegrims, for support on Docker issues, please contact them or @opsxcq who is the maintainer of the Docker image and repo. Any issue tickets will probably be pointed at this and closed._
+These assume you are on a Debian based distro, such as Debian, Ubuntu and Kali. For other distros, follow along, but update the command where appropriate.
 
-- [dockerhub page](https://hub.docker.com/r/vulnerables/web-dvwa/)
-`docker run --rm -it -p 80:80 vulnerables/web-dvwa`
+### Containers
 
-Please ensure you are using aufs due to previous MySQL issues. Run `docker info` to check your storage driver. If it isn't aufs, please change it as such. There are guides for each operating system on how to do that, but they're quite different so we won't cover that here.
+#### I want to access the logs
 
-- - -
+If you are using Docker Desktop, logs can be accessed from the graphical application.
+Some minor details may change with newer versions, but the access method should be the same.
 
-## Troubleshooting
+![Overview of DVWA compose](./docs/graphics/docker/overview.png)
+![Viewing DVWA logs](docs/graphics/docker/detail.png)
 
-These assume you are on a Debian based distro, such as Debian, Ubuntu and Kali. For other distros, follow along, but update the command where appropriate.
+Logs can also be accessed from the terminal.
+
+1. Open a terminal and change its working directory to DVWA
+2. Show the merged logs
+
+    ```shell
+    docker compose logs
+    ```
+
+   In case you want to export the logs to a file, e.g. `dvwa.log`
+
+   ```shell
+   docker compose logs >dvwa.log
+   ```
+
+#### Port 80 is busy because I am already running something on it
+
+To workaround this, you can expose DVWA on a different port by changing the port binding in the `compose.yml` file.
```

Comment:
As this is a file in the repo, if you make a change to it git will flag it as modified, if you then try to do a pull you could get conflicts which will stop the update from happening. Two things will then happen:

People don't read error messages so they will think they've updated and forever be stuck on an old version.
Someone will try to fix it and break their own repo which is frustrating.

Either way, tickets will be opened for it or because of it.

Reply:
Do you have any suggestions? Can we tell the user to turn on whatever is running on port 80, or just change the port from the beginning?

Reply:
Just change it from the beginning, but make it very clear that it is on a different port. If you can put up a message once the container comes up, do that, and include the full URL in it.